### PR TITLE
cloudflare-quiche 0.22.0

### PR DESCRIPTION
**Pull request by junhochoi**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by junhochoi** [comment]
CI fixed (linux support added)

**Comment by chenrui333** [comment]
@junhochoi 👋  thanks for the PR, just amended with commit msg fix. :)

**Comment by junhochoi** [comment]
> @junhochoi 👋 thanks for the PR, just amended with commit msg fix. :)

No problem! Thanks 👍

**Review by chenrui333 — APPROVED** [review]

File: Formula/c/cloudflare-quiche.rb [hunk 1]
```diff
@@ -23,10 +23,21 @@ class CloudflareQuiche < Formula
   def install
     system "cargo", "install", *std_cargo_args(path: "apps")
 
-    system "cargo", "build", "--lib", "--features", "ffi,pkg-config-meta", "--release"
-    lib.install "target/release/#{shared_library("libquiche")}"
+    system "cargo", "build", "-j", "1", "--lib", "--features", "ffi,pkg-config-meta", "--release"
```
Comment (chenrui333) [inline Formula/c/cloudflare-quiche.rb:26]: why this change?

**Review by chenrui333 — APPROVED** [review]

File: Formula/c/cloudflare-quiche.rb (hunk 1 above)
Comment (junhochoi) [reply Formula/c/cloudflare-quiche.rb:26]: oh this was only for debugging. need to be removed.

File: Formula/c/cloudflare-quiche.rb [hunk 2]
```diff
@@ -23,10 +23,21 @@ class CloudflareQuiche < Formula
...
+    # install dylib with version and symlink
+    lib.install "target/release/#{shared_library("libquiche")}"
+    if OS.mac?
+      mv "#{lib}/libquiche.dylib", "#{lib}/libquiche.#{version.major_minor_patch}.dylib"
+      lib.install_symlink "#{lib}/libquiche.#{version.major_minor_patch}.dylib" => "#{lib}/libquiche.dylib"
+    else
+      mv "#{lib}/libquiche.so", "#{lib}/libquiche.so.#{version.major_minor_patch}"
+      lib.install_symlink "#{lib}/libquiche.so.#{version.major_minor_patch}" => "#{lib}/libquiche.so.#{version.major}"
+      lib.install_symlink "#{lib}/libquiche.so.#{version.major_minor_patch}" => "#{lib}/libquiche.so"
+    end
```
[the remark this answers is not in the document]
Comment (junhochoi) [reply Formula/c/cloudflare-quiche.rb:39]: can you show me some example?
